### DISCUSSION TITLE
After successful "Save", the user sees all pages

### DISCUSSION THREAD

**Discussion by dcplebranch** [trendyminds/isolate#38] [Q&A]
User is only able to edit "Page One" so that's all they see when they click "Entries" initially... 

However, after user makes edits to that page and clicks "Publish", they are sent to a page that lists ALL entries and not just the isolated entries.

**Comment by wearealloy** [comment]
We are having the same issue. I will take a look at the code and see if I can find a fix. It seems that the fix should be to redirect the user to the Isolate dashboard after they have saved the entry.

**Comment by aaronbushnell** [comment]
Hello! We've moved all open Issues to **Discussions**.

Unfortunately, due to client workload we often find ourselves unable to implement new features or resolve issues in a timely manner. Switching to Discussions allows better categorization to separate "Feature Requests" from bugs. It also allows the community to better suggest/provide solutions when we're unable to do so.

Hope Isolate continues to be useful for you!

**Comment by patbrumfield** [comment]
I'm also experiencing this behavior, and curious if anyone else has figured out a solution? I'm also looking to have editors sent to the Isolate dashboard after saving an existing entry. Thoughts?
